ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/DatePhraseTd.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+
+import { getDateTime, getDistanceStrictAsPhrase } from 'utils/dateUtils';
+
+export type DatePhraseTdProps = {
+    date: string | number | Date | null | undefined;
+};
+
+function DatePhraseTd({ date }: DatePhraseTdProps) {
+    return (
+        <Tooltip content={getDateTime(date)}>
```

Comment:
shouldn't we check if `date` exists before passing it through to these functions since it can be one of 5 types?